test/e2e/performanceprofile/functests/README.txt

```diff
@@ -1,5 +1,16 @@
-HEADS UP!
+Functest consist in diffent suites:
 
-We have 2 test suites here, one running the "normal" performance tests, the other one performance profile update tests.
-The latter should run AFTER the former, and tests are executed in order of filenames.
-So be careful with renaming existing or adding new suites.
\ No newline at end of file
+0_config should be run before other in order to deploy PerformanceProfile and wait for MCP to pick MC and begin in updated state
+This deployment takes some time and requires reboot
+1- Performance performance test - should run after config and before update ones
+2- Performance update - Update test, should run after performance ones. 
+Different PerformanceProfiles and consequent updates to check if changes has been applied correclty
+3- Performance status - Check the status of differnet objects related with PAO
+4- Latency  // TODO
```

Comment:
4- Latency: runs tests of latency measurement tools and verifies the measurements are in the expected range according to the values of the latency environment variables.

Reply:
note also that the tests in this suite will be skipped unless LATENCY_TEST_RUN is set to true:
https://github.com/openshift-kni/performance-addon-operators#latency-test

Reply:
this is great information! do we want to have this level of detail here in this section?

Reply:
Well, maybe it would be nice to have a note on that. I feel this kind of information could save lot of time in the future.